src/useConfirm.js

```diff
@@ -11,7 +11,7 @@ const useConfirmId = () => {
   return `confirm-${id}`;
 };
 
-const useConfirm = () => {
+const useConfirm = (shouldCloseOnParentUnmount = true) => {
```

Comment:
Let's make this an option like all the others (passed to `confirm` call, or to the provider as defaults). See #94 for another PR adding options.

Reply:
Also, let's call the option `closeOnParentUnmount` :)